tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -80,6 +81,18 @@ void test_schwarzschild() {
   Approx custom_approx = Approx::custom().epsilon(1.e-12).scale(1.0);
   CHECK_ITERABLE_CUSTOM_APPROX(
       get(residual), DataVector(get(residual).size(), 0.0), custom_approx);
+
+  const auto unit_normal_vector = raise_or_lower_index(unit_normal_one_form,
+                                                       inverse_spatial_metric);
+  const auto ricci_scalar = StrahlkorperGr::ricci_scalar(
```

Comment:
After adding the additional helper functions, you can move this into a separate function so you can have separate test cases for each of your functions

Reply:
I did create a separate test case for each tested function.